src/screens/SelectCurrency.tsx

```diff
@@ -0,0 +1,65 @@
+import { Currency } from "@notuslabs/wallet";
+import { View, StyleSheet } from "react-native";
+
+import Container from "@/components/Container";
+import ActionButton from "@/components/buttons/ActionButton";
+import WalletContent from "@/components/buttons/WalletContent";
+import Heading from "@/components/typography/Heading";
+import { spacing } from "@/constants/DesignTokens";
+import useGetCurrenciesBalance from "@/hooks/useGetCurrenciesBalance";
+
+export type SelectCurrencyEventType = {
```

Comment:
Same thing about the suffix `Type`, I guess it's not needed here, it's an event type. Even javascript does not use it when defining event types, some examples:

1. `CustomEvent`
2. `MouseEvent`
3. `FormEvent`
4. ...

```suggestion
export type SelectCurrencyEvent = {
```